### ISSUE TITLE
Organization

### ISSUE THREAD

**Issue by melrobin** [shaifsalehin/COSC3342-Socket-Programming#2]
You have a lot of good code.  I would encourage you to organize your code into functions instead of a monolithic main function.  For instance your send code could be wrapped up.  This way you could preserve how you call the send function from main, but you implement sending in a different way.  

This would make your code more robust and nimble for possible implementation changes.

**Comment by shaifsalehin** [comment]
Yes, sir, I will look at putting more of my code into functions. 

What do you mean by the send code, its already a function called send_data, would you prefer I implemented it some other way?

**Comment by shaifsalehin** [comment]
Ah, after talking with you I have a better idea. I will work on making send more abstract and reducing the code on main function.
